src/user/supla_esp_devconn.c

```diff
@@ -25,6 +25,7 @@
 #include <osapi.h>
 #include <mem.h>
 #include <stdlib.h>
+#include <sntp.h>
```

Comment:
Do not change the file
src / user / supla_esp_devconn.c

all you need to move to the area of ​​your own board.